starfish_py/agent.py

```diff
@@ -23,12 +23,14 @@ def register(self, service_name, endpoint_url, account, did=None):
 
         Register an agent on the block chain. This is called by the method :func:`starfish_py.ocean.Ocean.register_agent`.
 
-        :param service_name: service_name to add into the DDO service field
-        :param endpoint_url: url of the agent service
-        :param account: Ethereum account to use to register on the block chain
-        :param did: optional did to update the did with this value (if you are the account owner of the DID)
-
-        :return: a list of DID, DDO and  private pem of the registered DDO.
+        :param str service_name: service_name to add into the DDO service field
+        :param str endpoint_url: url of the agent service
+        :param object account: Ethereum account to use to register on the block chain
+        :param did: Optional did to update the did with this value (if you are the account owner of the DID)
+        :type did: str or None
+
+        :return: a tuple of [ DID, DDO, private pem ] used in the registeration of a DDO.
```

Comment:
Use _()_  instead of _[]_ for the return type comment as _round brackets_ indicate a tuple but _square brackets_ indicate a list.